### ISSUE TITLE
3D Mode: Invalid asm.js: Invalid member of stdlib

### ISSUE THREAD

**Issue by Ysurac** [Ysurac/FlightAirMap#356]
Seems to be an error seen in Chrome/Chromium 61.x
Issue open on Cesium: https://github.com/AnalyticalGraphicsInc/cesium/issues/5847

**Comment by adriweb** [comment]
FWIW, this is apparently caused by Chrome now having that `chrome://flags/#enable-asm-webassembly` enabled by default.

```
Experimental Validate Asm.js and convert to WebAssembly when valid. Mac, Windows, Linux, Chrome OS, Android
Validate Asm.js when "use asm" is present and then convert to WebAssembly. #enable-asm-webassembly
```
Disabling it makes the message go away.

I found that out [here](https://chromium.googlesource.com/chromium/src/+/2b2ec4ad531ae43a75973857fb969d18d81a22c3%5E%21/#F0) randomly trying to understand (to no avail yet) why Chrome 62 is making my multi-threaded (2 threads) emulator very slow while it's working just fine on Chrome 61 and other latest browser versions...

**Comment by Ysurac** [comment]
Should be fixed since update to Cesium 1.40
